whipper/common/mbngs.py

```diff
@@ -161,10 +161,84 @@ def _getWorks(recording):
     if 'work-relation-list' in recording:
         for work in recording['work-relation-list']:
             if work['type-id'] in valid_work_rel_types:
-                works.append(work['work']['id'])
+                works.append(work['work'])
     return works
 
 
+def _getWorksId(works):
+    """Get work ids from "performance of" works"""
+    ids = []
+    for work in works:
+        ids.append(work['id'])
+    return ids
+
+
+def _getComposers(works):
+    """
+    Extract composer(s) from works' artist-relation-list and return a
+    string in format "A, B and C"
+
+    :rtype: str
+    """
+    res = ''
+    composers = []
+    valid_artist_rel_type = [
+        'd59d99ea-23d4-4a80-b066-edca32ee158f',  # "Composer"
+    ]
+    for work in works:
+        if 'artist-relation-list' in work:
+            for artist_relation in work['artist-relation-list']:
+                if artist_relation['type-id'] in valid_artist_rel_type:
+                    composerName = artist_relation['artist']['name']
+                    composers.append(composerName)
+    if len(composers) > 1:
+        res = ', '.join(composers[:-1]) + ' and ' + composers[-1]
```

Comment:
Please don’t. FLAC tags can contain multiple values, so just passing `composers` around as a list is fine.